### PR TITLE
Adding amazon/google to adoptions

### PR DESCRIPTION
**Pull request by jhdalek55**
Added the logos and links for these two adoptions--if I did this correctly. Could someone check before merging.

### CODE REVIEW

File: adoptions.md [hunk 1]
```diff
@@ -10,11 +10,18 @@ By clicking on the images below, you will be linked either to articles that
 cover the TUF adoption, presentations on the adoption, or to repositories
 containing the relevant code.
 
+<a
+href="https://github.com/awslabs/tough"
+rel="AWS Labs logo"><img src="/assets/images/amazon_logo.png"
+width="200"></a>
+<a href="https://fuchsia.dev/"
+rel="Google Fuchsia logo"><img src="/assets/images/Google_fuchsia3_logo.png"
```
Comment (JustinCappos) [inline adoptions.md]: Let's use the Google logo instead.  It's more recognizable.

File: adoptions.md [hunk 2]
```diff
@@ -23,17 +30,17 @@ rel="Microsoft Azure logo"><img src="/assets/images/azure_logo.jpg"
 width="200"></a>
 
 <a href="https://blog.docker.com/2015/08/content-trust-docker-1-8/" rel="docker
-logo"><img src="/assets/images/docker_logo.png" width="200"></a> 
+logo"><img src="/assets/images/docker_logo.png" width="200"></a>
 <a href="https://www.youtube.com/watch?v=2ZIu9XTvsC0"
 rel="VMware logo"><img src="/assets/images/vmware_logo.png"
-width="200"></a> 
+width="200"></a>
```
Comment (JustinCappos) [inline adoptions.md]: This likely was an error.  Can you remove unintended edits like these?

File: adoptions.md (hunk 1 above)
Comment (jhdalek55) [reply adoptions.md]: Ok. I can do that.

File: adoptions.md (hunk 2 above)
Comment (jhdalek55) [reply adoptions.md]: I may be missing something here but I don't see what needs fixing here.

File: adoptions.md (hunk 1 above)
Comment (jhdalek55) [reply adoptions.md]: OK. Logo substitution done.

**Comment by lukpueh** [comment]
Just reverted unrelated whitespace modifications and squashed @jhdalek55's two commits.

**Pull request merged**